addons/sale/views/product_pricelist_item_views.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="product_pricelist_item_form_view_inherit_product" model="ir.ui.view">
+    <field name="name">product.pricelist.item.view.form.inherit</field>
+    <field name="model">product.pricelist.item</field>
+    <field name="inherit_id" ref="product.product_pricelist_item_form_view"/>
+    <field name="arch" type="xml">
+        <group name="pricelist_rule_limits" position="replace">
+            <div>
+                <group name="pricelist_rule_limits">
+                    <field name="min_quantity" string="Min Qty"/>
+                    <field
+                        name="date_start"
+                        string="Validity Period"
+                        widget="daterange"
+                        options="{'end_date_field': 'date_end'}"
+                    />
+                    <field name="date_end" invisible="1"/>
+                </group>
+                <div groups="sale.group_discount_per_so_line">
+                    <div
+                        class="alert alert-info"
+                        role="alert"
+                        invisible="compute_price != 'percentage'"
+                    >
+                        <div>
+                            In sale order line original price is unit price and discount is in
+                            discount column.
+                        </div>
+                        <div>On the website original price is crossed out.</div>
+                    </div>
+                    <div
+                        class="alert alert-danger"
+                        role="alert"
+                        invisible="compute_price == 'percentage'"
+                    >
+                        <div>
+                            When prices are calculated through formula or fixed, original price is NOT
+                            displayed on website or in sale order lines. Calculated/fixed price is set as
+                            unit price with 0% discount
```

Comment:
Wrong, see `_show_discount_on_shop` logic, we also show the discount for some specific formula rules, but only on the shop, not on the cart/order.

Reply:
It's okay, was more an issue from our side TBH, I'm just telling you here because I noticed.